shared/teams/common/enable-contacts.tsx

```diff
@@ -28,9 +28,24 @@ const EnableContacts = ({onClose}: {onClose: () => void}) => {
   )
 }
 
+export const useEnableContactsPopup = (noAccess: boolean, onClose: () => void) => {
```

Comment:
As we discussed, let's switch this to be a component instead of a hook.

Also, can you add a js doc comment to it explaining what it does, I think its props are non-obvious.